whd/webhook.py

```diff
@@ -39,6 +45,14 @@ def __init__(
         self.dispatcher = GithubWebhookDispatcher(cfg_set=cfg_set, whd_cfg=whd_cfg)
 
     def post(self):
+        # resource checking can take long, so we execute in an own thread
```

Comment:
please state _why_ it's a problem the updating tennds to take long (btw, it is also true to state that any subsequent operations take potentially too long - it does not matter which at this point), namely we want to mark github-webhooks as "delivered successfully" even though we might not succeed in performing all tasks we want / need to upon the webhook request, so that we can identify actual delivery issues from github-reporting

Reply:
all done